man/last.Rd

```diff
@@ -18,6 +18,9 @@ of \code{xts::first} is deployed. }
 \item{\dots}{ Not applicable for \code{data.table} first/last. Any arguments here
 are passed through to \code{xts}'s first/last. }
 }
+\details{
+Note: For zero-length vectors, \code{first(x)} and \code{last(x)} mimic \code{head(x, 1)} and \code{tail(x, 1)} by returning an empty vector instead of \code{NA}. However, unlike \code{head()}/\code{tail()} and base R subsetting (e.g., \code{x[1]}), they do not preserve attributes like names.
```

Comment:
there is a Rd keyword `note` that seems more feasible here rather than `details`, considering sentence starts with "Note:".
example: https://github.com/Rdatatable/data.table/blob/ca65511cc0f91c3a73ecf129a4b8e444609a14ac/man/like.Rd#L33